### ISSUE TITLE
No more scrolling?

### ISSUE THREAD

**Issue by DeLordsManjunk** [ArcadeRenegade/SidebarDiagnostics#371]
**1. Is this a feature request?** yes and no i think

*If yes then sorry but this project is not taking feature requests at this time.*

**2. Your Operating System**  win 10

**3. App Version** latest 

**4. Any Relevant Settings** nope

**5. Steps to Reproduce**  I have a 5900x and of course all the CPU cores are showing.  I also have 2 SSD and 3 HDD then 3 internet one is WiFi 6 and other 2.5Gb LAN connection  and a 1Gb LAN connection.  Most times i only use the 2.5Gb connection but i have them all set to show if connected.  

Problem i am having is that i can't seem to find how to allow scrolling of the sidebar so that i can see each item.  What i have to do now is go in and change the settings for the cores to off and remove GPU and ram completely and i can see most of the other information.  If i recall right i used to be able to scroll up and down on the sidebar to see everything.  Did this feature get dropped or just stop working or was i wrong and it never existed?

Not sure when It really stopped scrolling.  Thanks for any help you can give or fixes that can be done.

**Comment by ArcadeRenegade** [comment]
Hi, the scrolling should still work if the content is taller than the sidebar. What version of Win 10 are you on?

**Comment by DeLordsManjunk** [comment]
21h1 1903.1165

**Comment by DeLordsManjunk** [comment]
A few things i have done but got it fixed now.  So i went through the settings one by one.  And that click through i didn't remember having on.  But it was checked i unchecked it and applied but no change.  Then i saved it and no change.  So i checked everything else and it still didn't work.  I closed the program and then restarted it and still no change when it first came back up.  After a few minutes of waiting i just happened to hover over it and it started to scroll. The only thing i changed was the click through.  So that had to be it.  But why did it take so long to start working?  So i turned it back on and right after i did, scroll stopped which was expected.  i saved then changed it back to off and applied again, applied didn't change it, saved didn't change it.  But after a few minutes it worked again.  So next what i did was move it back to screen one from screen two.  This time redid the same steps and it worked right after i turned it back off but no on the apply just the save.    So maybe it is stuck in memory for a moment?  Either way it is working now.  I guess i just need more patience lol.   I know you don't want feature request but why don't we have the temp of the drives or ram?  I know not really needed but would be nice to see while gaming.

**Comment by ArcadeRenegade** [comment]
Oh ya I think turning on click through makes it un-interactible with the mouse. So not being able to scroll might be a side effect of that. To be honest that wasn't fully tested, just something thrown in there at someone's request. Strange that the settings took so long to take effect though.

**Comment by ArcadeRenegade** [comment]
Drive temperatures isn't available in performance monitor in Windows.

**Comment by DeLordsManjunk** [comment]
Yeah that delay was what confused me.   If anyone else has this problem maybe it would be more concerning but if just me probably just my hardware or even another program.  I do run NZXT Cam also.  Usually to a small monitor i mounted in my case.  Maybe a conflict of resources calls on shared access of files.  Either way I am lost now a days when it comes to programming haven't do anything like that since 2000.    As for the drive temperatures makes since you are using the built in performance monitor to gather the info.   Again thanks for a great program.  I run it on 3 of my systems and yes just this one system had that issue the other 2 i checked later on today and were fine and no hang.  I'll drop my specs for you to see might help.


![Screenshot (13)](https://user-images.githubusercontent.com/49692430/134990407-af2f2dad-546c-4837-8ce7-f2ed76bf9feb.png)
